robottelo/rhsso_utils.py

```diff
@@ -24,9 +24,9 @@ def run_command(cmd, hostname=None, timeout=None):
         result = ssh.command(cmd=cmd, hostname=hostname, timeout=timeout)
     else:
         result = ssh.command(cmd=cmd, hostname=hostname)
```

Comment:
We could pass the host object, refactor this. We can remove this as well. let me know

Reply:
with the current design (until a much larger refactor) it is the same amount of time (if not faster) to pass along the hostname this way since run_command gets a host object from get_client which is lru_cached